sdk/core/core-tracing/src/useInstrumenter.ts

```diff
@@ -0,0 +1,14 @@
+import { NoOpInstrumenter } from "./instrumenter";
+import { Instrumenter } from "./interfaces";
+
+/** @internal */
+export let instrumenterImplementation: Instrumenter = new NoOpInstrumenter();
+
+/**
+ * Extends the Azure SDK with support for a given instrumenter implementation.
+ *
+ * @param instrumenter - The instrumenter implementation to use.
+ */
+export function useInstrumenter(instrumenter: Instrumenter): void {
+  instrumenterImplementation = instrumenter;
```

Comment:
In node the OTel instrumentation library handles patching any imports of @azure/core-tracing so we can ensure we always call useInstrumenter...